### ISSUE TITLE
[bug]: Regressing with pjsip trunk connection in the change #482

### ISSUE THREAD

**Issue by mjrider** [FreePBX/issue-tracker#519]
### FreePBX Version

FreePBX 17

### Issue Description

There is a issue with the pjsip trunk for the registration configuration

related to #482 

And it's related to this error
[2024-10-25 13:04:04] ERROR[1065511] config_options.c: Could not find option suitable for category 'freepbx-test' named 'allow_unauthenticated_options' at line 22 of /etc/asterisk/pjsip.registration.conf

- Doing a rollback to core version  17.0.18.1 the issue is resolved
- manual edit /etc/asterisk/pjsip.registration.conf and remove the allow_unauthenticated_options, it also resolved this

it doesnt matter if we use yes or no as option, both values are wrong

Also noted on https://github.com/FreePBX/issue-tracker/issues/482#issuecomment-2437557903_ because it is a regression
            

### Operating Environment

Debian bookworm

- asterisk version 21
- core 17.0.18.4

### Relevant log output

```shell
[2024-10-25 13:04:04] ERROR[1065511] config_options.c: Could not find option suitable for category 'freepbx-test' named 'allow_unauthenticated_options' at line 22 of /etc/asterisk/pjsip.registration.conf
```

**Comment by kapilgupta01** [comment]
Hi @mjrider can you please share the asterisk cli log file and generated configuration for us to review? 
please hide/remove any password or confidential data

**Comment by ramarajan222** [comment]
Fixed on latest core v17.0.18.5

**Comment by ramarajan222** [comment]
Hi @mjrider  Please update the core version to 17.0.18.5. thanks.

**Comment by mjrider** [comment]
sorry didn't added enough log data, thx that it is fixed
core 17.0.18.5 does work as expected

**Comment by kapilgupta01** [comment]
yes just now fixed and pushed core v17.0.18.5

thanks for confirming @mjrider
